dashboard/app/models/user.rb

```diff
@@ -816,6 +816,8 @@ def presence_of_email_or_hashed_email
   def email_and_hashed_email_must_be_unique
     # skip the db lookup if we are already invalid
     return if errors.present?
+    # allow duplicate accounts to be created for LMS users that are unlinked
+    return if authentication_options.length == 1 && authentication_options.first&.credential_type == AuthenticationOption::LTI_V1
```

Comment:
AuthenticationOption has an `lti?` method you can use like:

```
return if authentication_options.length == 1 && authentication_options.first&.lti?
```

Reply:
Is it possible for an account to having just an LTI auth option and then get another auth option (of any kind) which would suddenly forever invalidate the User?